pkg/plugins/policies/meshtrafficpermission/xds/rbac_configurer_test.go

```diff
@@ -252,5 +252,35 @@ filters:
       rules: {}
       statPrefix: no.`,
 		}),
+		Entry("shadow deny rule", testCase{
+			stats: "shadow_deny_prefix",
+			mesh:  "shadow_deny_mesh",
+			rules: []*core_xds.Rule{
+				{
+					Subset: []core_xds.Tag{
+						{Key: "kuma.io/service", Value: "backend"},
+					},
+					Conf: &v1alpha1.MeshTrafficPermission_Conf{
+						Action: "ALLOW_WITH_SHADOW_DENY",
+					},
+				},
+			},
+			expected: `
+filters:
+  - name: envoy.filters.network.rbac
+    typedConfig:
+      '@type': type.googleapis.com/envoy.extensions.filters.network.rbac.v3.RBAC
+      rules:
+          policies:
+              MeshTrafficPermission:
+                  permissions:
+                      - any: true
+                  principals:
+                      - authenticated:
+                          principalName:
+                              exact: spiffe://shadow_deny_mesh/backend
+      shadowRules: {}
```

Comment:
I'm lost here. What does empty `shadowRules` mean?

I'd expect that we will shadow deny only `kuma.io/service: backend`

Reply:
Right, I had in mind that the default action for shadow rules is `DENY` as we do with normal rules. But that was wrong assumption, I'll fix it